mlflow/models/container/__init__.py

```diff
@@ -31,10 +31,10 @@
 
 DEPLOYMENT_CONFIG_KEY_FLAVOR_NAME = "MLFLOW_DEPLOYMENT_FLAVOR_NAME"
 
-DEFAULT_SAGEMAKER_SERVER_PORT = 8080
-DEFAULT_INFERENCE_SERVER_PORT = 8000
-DEFAULT_NGINX_SERVER_PORT = 8080
-DEFAULT_MLSERVER_PORT = 8080
+DEFAULT_SAGEMAKER_SERVER_PORT = int(os.environ.get('DEFAULT_SAGEMAKER_SERVER_PORT', 8080))
```

Comment:
These are constants. The logic for accessing environment variables should be handled at the point of usage.